### ISSUE TITLE
[Question] Postfix and Dovecot delivery integration

### ISSUE THREAD

**Issue by tyranron** [tomav/docker-mailserver#305]
According to current (release 2.0.0) configuration Postfix delivers mails to account's mailboxes via LDA command:

[/target/postfix/main.cf:64-65](https://github.com/tomav/docker-mailserver/blob/2.0.0/target/postfix/main.cf#L64-L65)
```
# Enable Dovecot Sieve
virtual_transport = dovecot
```

[/target/postfix/master.cf:66-71](https://github.com/tomav/docker-mailserver/blob/2.0.0/target/postfix/master.cf#L66-L71)
```
#
# Dovecot LDA configuration
#

dovecot unix    -       n       n       -       -      pipe
  flags=DRhu user=docker argv=/usr/lib/dovecot/deliver -f ${sender} -d ${user}@${nexthop} -m ${extension}
```

So Postfix uses `/usr/lib/dovecot/deliver` util to deliver message to Dovecot. This applies some additional requirements, i.e. Postfix and Dovecot should have access to this directory under the same user and etc.

My question is:
Why not use LMTP over unix socket as [this page](http://wiki.dovecot.org/LDA) recommends?
LMTP really simplifies configuration and decreases processes coupling.
Do I miss any caveats?

**Comment by Josef-Friedrich** [comment]
Sounds interessting

**Comment by tomav** [comment]
Hi @tyranron, thanks for this input.
@00angus, any comment?

**Comment by 00angus** [comment]
lmtp! Very good IMHO.
I was for lmtp even before:
https://github.com/tomav/docker-mailserver/pull/166

Just my 0,02€ :)

-- Marco

On Sep 4, 2016 20:42, "Thomas VIAL" <notifications@github.com> wrote:

Hi @tyranron <https://github.com/tyranron>, thanks for this input.
@00angus <https://github.com/00angus>, any comment?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/tomav/docker-mailserver/issues/305#issuecomment-244621325>,
or mute the thread

**Comment by Josef-Friedrich** [comment]
Why still hesitate? @tyranron can you implement LMTP in `docker-mailserver`?

**Comment by tyranron** [comment]
@Josef-Friedrich yeap, I'll try in next few days.

**Comment by tomav** [comment]
Hi @tyranron, any update on this? Thank you.

**Comment by tyranron** [comment]
@tomav oops, sorry. Totally forget about this. It was better to say "few months" =/

**Comment by tomav** [comment]
Fixed by #360, thanks @tyranron
